docs/concepts/logs.md

```diff
@@ -125,6 +125,42 @@ Prefect automatically uses the task run logger based on the task context. The de
 
 The underlying log model for task runs captures the task name, task run ID, and parent flow run ID, which are persisted to the database for reporting and may also be used in custom message formatting.
 
+### Logging print statements
+
+For prototyping or local debugging, Prefect provides a decorator kwarg `log_print=True` to log print statements at the task or flow level. Tasks and subflows will inherit the print logging behavior from their parent flow.
```

Comment:
```suggestion
For prototyping or local debugging, Prefect provides the `log_prints` option to enabling logging of print statements at the task or flow level. By default, tasks and subflows will inherit the print logging behavior from their parent flow.
```

Do we want to have the prototyping/debugging note?

Reply:
"to enable logging of" I think?